src/components/Auth/AuthNavigator.js

```diff
@@ -5,8 +5,8 @@ import wrapNavigator, { routeKeys } from 'components/shared/wrapNavigator';
 import Loading from './Loading';
 import Mnemonic from './Mnemonic';
 import PinSetup from './PinSetup';
-import Ready from './Ready';
 import Recovery from './Recovery';
+import Choose from './PinSetup/PinPad/Choose';
```

Comment:
What `Auth/Choose` route stands for? I can not see any actual routing to that screen